R/pkg/R/SQLContext.R

```diff
@@ -37,6 +37,37 @@ getInternalType <- function(x) {
          stop(paste("Unsupported type for SparkDataFrame:", class(x))))
 }
 
+#' Temporary function to reroute old S3 Method call to new
+#' We need to check the class of x to ensure it is SQLContext before dispatching
+dispatchFunc <- function(newFuncSig, x, ...) {
```

Comment:
I think this is very specific to this file - this helper is implemented to specifically check & remove sqlContext parameter; I'll add more documentation on this.